### PR TITLE
PMM-9803 incorrect connection message

### PR DESCRIPTION
**Pull request by filipmikes1**
**What this PR does / why we need it**: Fixing confusing message shown for PMM users already connected to Platform

**Which issue(s) this PR fixes**: https://jira.percona.com/browse/PMM-9803

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: public/app/percona/shared/components/Elements/PlatformConnectedLoader/PlatformConnectedLoader.tsx [hunk 1]
```diff
@@ -5,23 +5,36 @@ import { Messages } from './PlatformConnectedLoader.messages';
...
   const styles = useStyles(getStyles);
-  const featureSelector = useCallback((state: StoreState) => !!state.perconaUser.isPlatformUser, []);
+  const featureSelector = useCallback((state: StoreState) => !!state.perconaSettings.isConnectedToPortal, []);
+  const { isPlatformUser } = useSelector(getPerconaUser);
+
+  const checkForPlatformUser = () => {
+    if (isPlatformUser) {
+      return children;
+    }
+    return <EmptyBlock dataTestId="not-platform-user">{Messages.platformUser}</EmptyBlock>;
```
Comment (fabio-silva) [inline public/app/percona/shared/components/Elements/PlatformConnectedLoader/PlatformConnectedLoader.tsx:21]: Can we use `useCallback` for this?

File: public/app/percona/shared/components/Elements/PlatformConnectedLoader/PlatformConnectedLoader.tsx [hunk 2]
```diff
@@ -5,23 +5,36 @@ import { Messages } from './PlatformConnectedLoader.messages';
...
+  };
+
+  const ErrorMessage = () => (
```
Comment (fabio-silva) [inline public/app/percona/shared/components/Elements/PlatformConnectedLoader/PlatformConnectedLoader.tsx:24]: I think we should declare this component outside of here, as a component. Not necessarily a new file, but just declare it outside.

**Review by fabio-silva — CHANGES_REQUESTED** [review]

**Review by fabio-silva — APPROVED** [review]

File: public/app/percona/shared/components/Elements/PlatformConnectedLoader/PlatformConnectedLoader.tsx (hunk 2 above)
Comment (filipmikes1) [reply public/app/percona/shared/components/Elements/PlatformConnectedLoader/PlatformConnectedLoader.tsx]: I moved it to separate file to make it more consistent

File: public/app/percona/shared/components/Elements/PlatformConnectedLoader/PlatformConnectedLoader.tsx (hunk 1 above)
Comment (filipmikes1) [reply public/app/percona/shared/components/Elements/PlatformConnectedLoader/PlatformConnectedLoader.tsx:18]: Added

**Pull request merged**